### PR TITLE
added javadoc items for the java rest api wrappers

### PR DESCRIPTION
**Pull request by yvchan**
in addition to the javadoc, i've also removed any functions that should be returning structures instead of strings.  (i.e, things like getHost() or getInstance() and have left them out until we need to add them in

### CODE REVIEW

**Comment by ddebrunner** [comment]
Can one of the admins verify this patch?

File: java/src/com/ibm/streamsx/rest/Job.java [hunk 1]
```diff
@@ -105,128 +109,137 @@ void setConnection( final StreamsConnection sc) {
...
+     * @return {@link List} of {@link String}
+     */
+    public List<String> getSubmitParameters() {
         return submitParameters;
     }
 
+    /**
+     * Gets the Epoch time when this job was submitted
+     * 
+     * @return long
```
Comment (ddebrunner) [inline java/src/com/ibm/streamsx/rest/Job.java]: The `@return` needs to describe what is returned, not the type of the return.

**Comment by ddebrunner** [comment]
General comment that the `@return` clause needs to describe what is returned, not the type. The type is already described by the method signature.

Look at existing javadoc (e.g. Java SDK) to see typical styles.

**Comment by yvchan** [comment]
i've updated the @return clauses.

**Pull request merged**